artemis-features/src/main/resources/features.xml

```diff
@@ -54,8 +54,8 @@
 		<bundle dependency="true">mvn:org.jboss.logging/jboss-logging/${jboss.logging.version}</bundle>
 		<bundle dependency="true">mvn:org.jgroups/jgroups/${jgroups.version}</bundle>
 
-		<bundle dependency="true">mvn:org.apache.geronimo.specs/geronimo-json_1.0_spec/${json-p.spec.version}</bundle>
-		<bundle dependency="true">mvn:org.apache.johnzon/johnzon-core/${johnzon.version}</bundle>
+		<bundle dependency="true">mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.json-api-1.1/2.8-SNAPSHOT</bundle>
```

Comment:
@gnodet I'm about to merge this.. although I'm squashing all your committs as one.. (there's no need to keep individual committs).. 


although... for my education here..what's the difference between org.apache.geronimo.specs and servicemix? just wanted to know the difference.